ceph_deploy/hosts/fedora/__init__.py

```diff
@@ -18,7 +18,7 @@ def choose_init():
 
     Returns the name of a init system (upstart, sysvinit ...).
     """
-    return 'sysvinit'
+    return 'systemd'
```

Comment:
Making a conditional yum/dnf switchover seems fine.

The hope is to get these systemd changes merged this week for infernalis, which means that any community users will need the ceph-deploy smarts to handle sysvinit vs systemd.  :/